tmd/Tree/Tree.py

```diff
@@ -25,8 +28,8 @@ class Tree(object):
     from tmd.Tree.methods import get_multifurcations
     from tmd.Tree.methods import get_terminations
```

Comment:
same here, Tree is used always, thus scoped imports are not necessary here.

Reply:
I need to call these as methods, not functions.